### PR TITLE
UI: Draw Clipping Audio Indicator / Effect

### PR DESCRIPTION
**Pull request by Andersama**
UI: Draw Clipping Audio Indicator / Effect

Sets end portion of volumeter to a specified color for a length of time after clipped audio.
Effect lasts for 3 seconds by default.

### CODE REVIEW

**Comment by Fenrirthviti** [comment]
Could I trouble you to update the theme files to remove the extra whitespace and comment? I don't think it's necessary given that it's already with the rest of the volume meter color settings, and the property title is pretty clear what it is. This will keep the styling of the theme files consistent.

File: UI/volume-control.cpp [hunk 1]
```diff
@@ -832,7 +853,10 @@ void VolumeMeter::paintEvent(QPaintEvent *event)
 			5, channelNr * 4, width - 5, 3,
 			displayMagnitude[channelNr], displayPeak[channelNr],
 			displayPeakHold[channelNr]);
-
+
```
Comment (tjientavara) [inline UI/volume-control.cpp]: Please don't uncomment this. It is important for the little box in the front of the meter to be drawn for debugging audio problems.

**Comment by tjientavara** [comment]
Didn't you update this patch already on your own machine?

File: UI/volume-control.cpp (hunk 1 above)
Comment (sexmen130) [reply UI/volume-control.cpp]: No se como funciona i nose ni que e tocao

Comment (sexmen130) [reply UI/volume-control.cpp]: Que ago para arreglarlo

**Comment by Andersama** [comment]
Sorry I didn't get back to this sooner, I just pushed the blink feature and made the changes like you guys asked. The idle indicator's intact, but in order to make the draws not overwrite the clipping indicator I moved the idle draw into the main drawing section.

**Comment by sexmen130** [comment]
No speak inglish

**Comment by jp9000** [comment]
I prefer it to just hold for one second.  Also, I think QTimer::singleShot is probably a bit more simple for this case.

Additionally, the clip color is already available to theme files, and doesn't really need an additional property.

**Pull request closed without merging**